python/examples/basic/Getting_Started.ipynb

```diff
@@ -65,7 +65,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install -q whylogs --pre"
+    "! pip install -q whylogs --pre"
```

Comment:
It's a bit strange to have extra spaces here?

Reply:
but i can remove it